src/Compilers/CSharp/Test/Semantic/Semantics/ExpressionBodiedMemberTests.cs

```diff
@@ -481,28 +489,37 @@ public void BlockBodyAndExpressionBody_04()
             var comp = CreateCompilationWithMscorlib(@"
 public class C
 {
-    static int P1 {get; set;}
+    int P1 {get; set;}
 
     ~C()
-    { }
+    { P1 = 1; }
     => P1;
 }
 ");
 
             comp.VerifyDiagnostics(
-    // (8,5): error CS1519: Invalid token '=>' in class, struct, or interface member declaration
-    //     => P1;
-    Diagnostic(ErrorCode.ERR_InvalidMemberDecl, "=>").WithArguments("=>").WithLocation(8, 5),
-    // (8,10): error CS1519: Invalid token ';' in class, struct, or interface member declaration
-    //     => P1;
-    Diagnostic(ErrorCode.ERR_InvalidMemberDecl, ";").WithArguments(";").WithLocation(8, 10),
-    // (8,10): error CS1519: Invalid token ';' in class, struct, or interface member declaration
-    //     => P1;
-    Diagnostic(ErrorCode.ERR_InvalidMemberDecl, ";").WithArguments(";").WithLocation(8, 10)
+    // (5,5): error  CS8057: Methods cannot combine block bodies with expression bodies.
+    Diagnostic(ErrorCode.ERR_BlockBodyAndExpressionBody, @"~C()
+    { P1 = 1; }
+    => P1;").WithLocation(6, 5)
```

Comment:
Minor: Comment above "(5,5)" doesn't seem to line up with the `WithLocation(6,5)`